providers/factories/oracle/api.go

```diff
@@ -95,6 +96,8 @@ func APIQueryHandlerFactory(
 		apiPriceFetcher, err = raydium.NewAPIPriceFetcher(logger, cfg.API, metrics)
 	case providerName == osmosis.Name:
 		apiPriceFetcher, err = osmosis.NewAPIPriceFetcher(logger, cfg.API, metrics)
+	case providerName == polymarket.Name:
+		apiDataHandler, err = polymarket.NewAPIHandler(cfg.API)
```

Comment:
You're also going to need to add the default configuration to `cmd/constants/providers.go`. Would you also mind creating a sample mm just for the polymarket provider in `cmd/constants/marketmaps/markets.go`

Reply:
We should create a test deployment for this. feel free to ping once this is ready and I can set that all up.

Reply:
added [add polymarket to constants, reformat initialization](https://github.com/skip-mev/slinky/pull/629/commits/9cd4d6674c8ac64db56d19a56ea73fae497b1b77) and [add default provider to constsants](https://github.com/skip-mev/slinky/pull/629/commits/b69bf49f86a9d74990d0d169003f09fa6dac4b4d). 

will ping you once everyones spoken their peace on the PR